src/main/java/core/basesyntax/model/Shark.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.model;
+
+import core.basesyntax.interfaces.Swimmable;
+
+public class Shark extends Pisces implements Swimmable {
```

Comment:
```suggestion
public class Shark extends Pisces {
```
implements Swimmable is redundant, cause Pisces implements swimmable and swim is already overriden